src/lib/storage/dictionary_column/dictionary_encoder.hpp

```diff
@@ -25,6 +27,8 @@ class DictionaryEncoder : public ColumnEncoder<DictionaryEncoder> {
  public:
   static constexpr auto _encoding_type = enum_c<EncodingType, EncodingType::Dictionary>;
   static constexpr auto _uses_vector_compression = true;  // see base_column_encoder.hpp for details
+  // Allow usage of FixedString in the DictionaryColumn
+  static const auto _enable_fixed_strings = false;
```

Comment:
Wait - allow usage, but „enable“ is „false“?

Reply:
It looks to me as if the `FixedString` optimization is completely disabled in this PR and as if there is no interface to enable it. Is this correct?

Maybe this is an issue that belongs into #63 and there should be some global way to turn on/off this optimization. That‘s outside the scope of this PR though.